CLAUDE.md

```
@@ -304,6 +310,43 @@ NODE_ENV=development
 
 **Note:** The starmark.dev site should have a `.env` file with your Linear credentials for testing the storage connectors. We can add these as you provide them.
 
+## Code Review Process
+
+### CodeRabbit Integration
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)